### ISSUE TITLE
Maven repository

### ISSUE THREAD

**Issue by zepheiryan** [whirlycott/Whirlycache#1]
I was just searching the web to see what the state of Whirlycache was as it pertains to a project I work on here on GitHub and noticed you'd moved your code here just today.  Nice.  Welcome.  Are you considering publishing Whirlycache to a Maven repository?

**Comment by whirlycott** [comment]
Yes, I definitely will as soon as I do some long overdue housekeeping and stabilization!

**Comment by zepheiryan** [comment]
Great to hear - thanks for sharing the software with the world, it's been an excellent tool in the kit.

**Comment by tgkprog** [comment]
Used Whirly 2010-2013 again 2015. Please publish it to maven? Want to use again.  Do an as is version and improve it later?
